tests/test_mpi.py

```diff
@@ -12,10 +12,19 @@
                            retrieve_iteration_tree)
 from devito.mpi import MPI
 from devito.mpi.routines import HaloUpdateCall, MPICall
+from devito.mpi.distributed import CustomTopology, Distributor
 from examples.seismic.acoustic import acoustic_setup
 
 pytestmark = skipif(['nompi'], whole_module=True)
 
+class Dummy_input_comm():
```

Comment:
can get away with just a namedtuple but that's fine with me like that too

Reply:
also note the chosen class name ain't pythonic